src/DeviceProvisioning/ProvisioningWebApi/ProvisioningWebApi/App_Start/AutoFacConfig.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Autofac;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Web;
+using Autofac.Integration.WebApi;
+using System.Web.Http;
+using DeviceProvisioning.AccessTokens;
+using DeviceProvisioning.DeviceRegistry;
+
+namespace DeviceProvisioning
+{
+    public static class AutoFacConfig
```

Comment:
Ah... this will stir some conversation. I'm a fan though. I support this choice.